app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.kt

```diff
@@ -59,7 +59,11 @@ class MyDetailsOverviewRowPresenter(
 			}
 
 			binding.fdButtonRow.removeAllViews()
-			for (button in row.actions) binding.fdButtonRow.addView(button)
+			for (button in row.actions) {
+				if (button.parent != null) {
```

Comment:
This check makes the buttons never show at all.

Reply:
removed from "MyDetailsOverViewRow" all views; but without this check `if (button.parent != null)` I cant get rid off  -> `java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.`

Reply:
You're storing the views in the presenter instead of the viewholder right now. That means views are still re-used. You need to make sure to never re-use the views and scope them to the viewholder.